cms/static/cms/js/build.js

```diff
@@ -34,6 +34,7 @@
         modules: getModulesList([
             'js/factories/asset_index',
             'js/factories/base',
+            'js/factories/bulkupdate',
```

Comment:
Hrmm, is there no corresponding entry for `bulksettings` or the other utilities?

Reply:
I don't see any